src/runners/S3JUnitReporter.ts

```diff
@@ -0,0 +1,36 @@
+import { Callback } from 'aws-lambda';
+import { Credentials, S3 } from 'aws-sdk'
+import * as fs from 'fs';

```

Comment:
i'd import just what you are using, this may bloat the handlers that import this if using a wildcard import,.

Reply:
agree. removed credentials